invokeai/app/invocations/compel.py

```diff
@@ -65,23 +65,20 @@ def invoke(self, context: InvocationContext) -> CompelOutput:
             **self.clip.text_encoder.dict(),
         )
         with tokenizer_info as orig_tokenizer,\
-             text_encoder_info as text_encoder,\
-             ExitStack() as stack:
+             text_encoder_info as text_encoder:
 
-            loras = [(stack.enter_context(context.services.model_manager.get_model(**lora.dict(exclude={"weight"}))), lora.weight) for lora in self.clip.loras]
+            loras = [(context.services.model_manager.get_model(**lora.dict(exclude={"weight"})).context.model, lora.weight) for lora in self.clip.loras]
 
             ti_list = []
             for trigger in re.findall(r"<[a-zA-Z0-9., _-]+>", self.prompt):
                 name = trigger[1:-1]
```

Comment:
compel already returns loras as part of the `Conjunction` object if user adds `withLora(name, weight)` to the prompt - maybe use that instead of this?

Reply:
This can't be used as it can be applied only for one prompt, while we can have multiple prompts